### PR TITLE
Cleanup implementation

### PR DESCRIPTION
**Pull request by arkaitzgarro**
Clean up the implementation a little bit: remove unnecessary files, organise services, controllers, etc.

Feature wise, it creates a transfer with the provided files, and stores the data into the DB. It also retrieves the list of transfers on page load.

### CODE REVIEW

File: public/js/maps.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+})();
+
+
+
+
+
+
+
+
+// let selectedLocation = null;
```
Comment (bermannoah) [inline public/js/maps.js:70]: Will we need this later

**Review by bermannoah — APPROVED** [review]
❤️❤️❤️

File: services/wetransfer.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const https = require('https');
+const createWTClient = require('@wetransfer/js-sdk');
+
+async function createTransfer(name, description, files) {
+  const wtClient = await createWTClient(process.env.wt_api_key);
```
Comment (offinga) [inline services/wetransfer.js:5]: Can you maybe add an `example.env` file or something?

File: public/js/maps.js (hunk 1 above)
Comment (arkaitzgarro) [reply public/js/maps.js:70]: I was not sure about the next steps, so I removed everything that wasn't used.

**Comment by arkaitzgarro** [comment]
Closing because of https://github.com/bermannoah/maptransfer/pull/3

File: public/js/maps.js (hunk 1 above)
Comment (bermannoah) [reply public/js/maps.js:70]: kewl

**Pull request closed without merging**